### ISSUE TITLE
Error: More than one instance of bitcore-lib found

### ISSUE THREAD

**Issue by sc0Vu** [sc0Vu/vuethwallet#15]
Uncaught Error: More than one instance of bitcore-lib found. Please make sure to require bitcore-lib and check that submodules do not also include their own bitcore-lib dependency.
  at webpack:///~/bitcore-mnemonic/~/bitcore-lib/index.js:12:0 <- index.js:84076

See https://github.com/bitpay/bitcore/issues/1454

**Comment by JellyBool** [comment]
Hi @sc0Vu any progress ? I have the same issue here

**Comment by sc0Vu** [comment]
@JellyBool yup, I just remove the lightwallet package and using yoethwallet.

Will merge into master soon!
